tests/h/presenters_test.py

```diff
@@ -5,573 +5,335 @@
+            annotation=mock.Mock(
+                target_selectors=[{'exact': 'selected text'}],
+                text="entered text"
+            )
         )
-    )
-
-    title = annotation.title
-
-    assert jinja2.escape(spam_link) in title
-    for char in ['<', '>', '"', "'"]:
-        assert char not in title
-    assert isinstance(title, jinja2.Markup)
-
-
-@title_fixtures
-def test_title_with_file_uri(filename):
-    """If the document has no title and the annotation has a file:// uri then
-    it should return the filename part only."""
-    filename.return_value = "MyFile.pdf"
-    annotation = _annotation(mock.Mock(document=mock.Mock(title=None)))
-
-    assert annotation.title == "MyFile.pdf"
-
-
-@title_fixtures
-def test_title_returns_Markup_when_filename_returns_Markup(filename):
-    filename.return_value = jinja2.Markup("MyFile.pdf")
-    annotation = _annotation(mock.Mock(document=mock.Mock(title=None)))
-
-    assert isinstance(annotation.title, jinja2.Markup)
-
-
-@title_fixtures
-def test_title_unquotes_uris(uri, filename):
-    filename.return_value = ""  # This is not a file:// URI.
-    uri.return_value = "http://example.com/example%201.html"
-    annotation = _annotation(mock.Mock(document=mock.Mock(title=None)))
-
-    assert annotation.title == "http://example.com/example 1.html"
-
-
-@title_fixtures
-def test_title_returns_Markup_when_uri_returns_Markup(uri, filename):
-    filename.return_value = ""  # This is not a file:// URI.
-    uri.return_value = jinja2.Markup("http://example.com/example.html")
-    annotation = _annotation(mock.Mock(document=mock.Mock(title=None)))
-
-    assert isinstance(annotation.title, jinja2.Markup)
-
-
-@title_fixtures
-def test_title_when_document_has_None_for_title(uri, filename):
-    """If the document has None for its title it should use the uri instead."""
-    uri.return_value = "http://example.com/example.html"
-    filename.return_value = ""  # This is not a file:// URI.
-    annotation = _annotation(mock.Mock(document=mock.Mock(title=None)))
-
-    assert annotation.title == "http://example.com/example.html"
-
 
-@title_fixtures
-@pytest.mark.parametrize('annotation', [
-    mock.Mock(document=mock.Mock(title={'foo': 'bar'})),
-    mock.Mock(document=mock.Mock(title=[1, 2, 3]))])
-def test_title_when_document_title_is_not_a_string(uri, filename, annotation):
-    """If the document has None for its title it should use the uri instead."""
-    uri.return_value = u"http://example.com/example.html"
-    filename.return_value = ""  # This is not a file:// URI.
+        assert annotation.description == (
+            "&lt;blockquote&gt;selected text&lt;/blockquote&gt;entered text")
 
-    annotation = _annotation(annotation)
-    assert isinstance(annotation.title, unicode)
+    def test_created_day_string_from_annotation(self):
+        annotation = self._annotation(
+            annotation=mock.Mock(
+                created="2015-09-04T17:37:49.517852+00:00"))
```

Comment:
This should be `datetime.datetime.datetime(2015, 9, 4, 17, 37, 49, 517852)` now.